wahanegi/vibereport
# CI-551: [Snyk] Security upgrade axios from 1.4.0 to 1.6.0

[![CI-551](https://badgen.net/badge/JIRA/CI-551/0052CC)](https://cloverpop-internship.atlassian.net/browse/CI-551) [<img width="16" alt="Powered by Pull Request Badge" src="https://user-images.githubusercontent.com/1393946/111216524-d2bb8e00-85d4-11eb-821b-ed4c00989c02.png">](https://pullrequestbadge.com/?utm_medium=github&utm_source=wahanegi&utm_campaign=badge_info)

<p>This PR was automatically created by Snyk using the credentials of a real user.</p><br /><h3>Snyk has created this PR to fix one or more vulnerable packages in the `yarn` dependencies of this project.</h3>



#### Changes included in this PR

- Changes to the following files to upgrade the vulnerable dependencies to a fixed version:
    - package.json
    - yarn.lock


#### Note for [zero-installs](https://yarnpkg.com/features/zero-installs) users

If you are using the Yarn feature [zero-installs](https://yarnpkg.com/features/zero-installs) that was introduced in Yarn V2, note that this PR does not update the `.yarn/cache/` directory meaning this code cannot be pulled and immediately developed on as one would expect for a zero-install project - you will need to run `yarn` to update the contents of the `./yarn/cache` directory. 
If you are not using zero-install you can ignore this as your flow should likely be unchanged.


#### Vulnerabilities that will be fixed
##### With an upgrade:
Severity                   | Priority Score (*)                   | Issue                   | Breaking Change                   | Exploit Maturity
:-------------------------:|-------------------------|:-------------------------|:-------------------------|:-------------------------
![high severity](https://res.cloudinary.com/snyk/image/upload/w_20,h_20/v1561977819/icon/h.png "high severity")  |  **748/1000**  <br/> **Why?** Proof of Concept exploit, Recently disclosed, Has a fix available, CVSS 7.1  | Cross-site Request Forgery (CSRF) <br/>[SNYK-JS-AXIOS-6032459](https://snyk.io/vuln/SNYK-JS-AXIOS-6032459) |  No  | Proof of Concept 

(*) Note that the real score may have changed since the PR was raised.











Check the changes in this PR to ensure they won't cause issues with your project.



------------



**Note:** *You are seeing this because you or someone else with access to this repository has authorized Snyk to open fix PRs.*

For more information:  <img src="https://api.segment.io/v1/pixel/track?data=eyJ3cml0ZUtleSI6InJyWmxZcEdHY2RyTHZsb0lYd0dUcVg4WkFRTnNCOUEwIiwiYW5vbnltb3VzSWQiOiI3YTFkY2I1NS1iYzhlLTRjNjYtOWE4OC1mOGM0NmNkYTMxNmYiLCJldmVudCI6IlBSIHZpZXdlZCIsInByb3BlcnRpZXMiOnsicHJJZCI6IjdhMWRjYjU1LWJjOGUtNGM2Ni05YTg4LWY4YzQ2Y2RhMzE2ZiJ9fQ==" width="0" height="0"/>
🧐 [View latest project report](https://app.snyk.io/org/rogergraves/project/470fc4b7-727f-47cc-847e-19bd58068579?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr)

🛠 [Adjust project settings](https://app.snyk.io/org/rogergraves/project/470fc4b7-727f-47cc-847e-19bd58068579?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr/settings)

📚 [Read more about Snyk's upgrade and patch logic](https://support.snyk.io/hc/en-us/articles/360003891078-Snyk-patches-to-fix-vulnerabilities)

[//]: # (snyk:metadata:{"prId":"7a1dcb55-bc8e-4c66-9a88-f8c46cda316f","prPublicId":"7a1dcb55-bc8e-4c66-9a88-f8c46cda316f","dependencies":[{"name":"axios","from":"1.4.0","to":"1.6.0"}],"packageManager":"yarn","projectPublicId":"470fc4b7-727f-47cc-847e-19bd58068579","projectUrl":"https://app.snyk.io/org/rogergraves/project/470fc4b7-727f-47cc-847e-19bd58068579?utm_source=github&utm_medium=referral&page=fix-pr","type":"auto","patch":[],"vulns":["SNYK-JS-AXIOS-6032459"],"upgrade":["SNYK-JS-AXIOS-6032459"],"isBreakingChange":false,"env":"prod","prType":"fix","templateVariants":["updated-fix-title","priorityScore"],"priorityScoreList":[748],"remediationStrategy":"vuln"})


---
## How to test

Please check if the app and the leader panel are functioning correctly after an update.

---

**Learn how to fix vulnerabilities with free interactive lessons:**

 🦉 [Cross-site Request Forgery (CSRF)](https://learn.snyk.io/lesson/csrf-attack/?loc&#x3D;fix-pr)

[CI-551]: https://cloverpop-internship.atlassian.net/browse/CI-551?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
